bin/faucet/src/main.rs

```diff
@@ -56,50 +58,78 @@ pub struct Cli {
     pub command: Command,
 }
 
+#[allow(clippy::large_enum_variant)]
 #[derive(Subcommand)]
 pub enum Command {
     /// Start the faucet server
     Start {
-        #[arg(short, long, value_name = "FILE", default_value = FAUCET_CONFIG_FILE_PATH)]
-        config: PathBuf,
+        /// Endpoint of the faucet in the format `<ip>:<port>`.
+        #[arg(long = "endpoint", value_name = "URL", env = ENV_ENDPOINT)]
+        endpoint: Url,
+
+        /// Node RPC gRPC endpoint in the format `http://<host>[:<port>]`.
+        #[arg(long = "node-url", value_name = "URL", env = ENV_NODE_URL)]
+        node_url: Url,
+
+        /// Timeout for RPC requests in milliseconds.
+        #[arg(long = "timeout", value_name = "MILLISECONDS", default_value_t = 5000, env = ENV_TIMEOUT)]
+        timeout_ms: u64,
+
+        /// Path to the faucet account file.
+        #[arg(long = "account", value_name = "FILE", env = ENV_ACCOUNT_PATH)]
+        faucet_account_path: PathBuf,
+
+        /// Comma-separated list of amounts of asset that should be dispersed on each request.
+        #[arg(long = "asset-amounts", value_name = "U64", env = ENV_ASSET_AMOUNTS, num_args = 1.., value_delimiter = ',', default_value = "100,500,1000")]
```

Comment:
nit: I tend to prefer the typed variant: `default_value_t = vec![100,500,1000]` _unless_ it's hidious as hell, which it might be in this case (ref https://docs.rs/clap/latest/clap/_derive/index.html#arg-attributes )